samples/benchmarks/coremark/src/main.c

```diff
@@ -130,7 +130,7 @@ int main(void)
 			k_panic();
 		}
 
-		LOG_INF("Press %s to start the test ...",  BUTTON_LABEL);
+		LOG_INF("Press %s to start the test ...\n",  BUTTON_LABEL);
```

Comment:
this doesn't really improve readability, it looks really weird in the output

Reply:
This sample already uses break lines in other places, I wanted to add them here for consistency so that we have a line break before every wait period (e.g. `k_sem_take`).